governance/program/src/processor/process_deposit_governing_tokens.rs

```diff
@@ -91,6 +116,18 @@ pub fn process_deposit_governing_tokens(
         governing_token_owner_info.key,
     );
 
+    // Adjust deposit amount to account for transfer fees
+    // Fee calculation requires to be on-chain due to epoch clock requirement
+    // Ensures accurate token amount after fee deduction to be stored in the TokenOwnerRecord
+    let deposit_amount = match expected_mint_info {
+        Some(mint_info) => {
+            amount.checked_sub(get_current_mint_fee(mint_info, amount)?).unwrap()
```

Comment:
note sure, but do we want to allow ".unwraps" ?

Reply:
this is done.